js/function.js

```diff
@@ -0,0 +1,53 @@
+// Функция для проверки длины строки
+
+const searchStringValidation = (test, number) => {
+  const lenght = test.length >= number;
```

Comment:
Опечатка в length. Здесь так же название не совсем отражает суть. "Длина" подразумевает что там будет число, а у нас булево значение, необходимо заменить. В целом вводить эту доп. переменную можно не вводить и сразу сделать возврат 
```suggestion
  return test.length >= number;
```